dask_chtc/cli.py

```diff
@@ -430,7 +430,8 @@ def contact_address(self) -> str:
     def start(self, jupyter_args: List[str]) -> "JupyterJobManager":
         if self.has_running_job():
             raise click.ClickException(
-                'You already have a running Jupyter notebook server; try the "status" subcommand to see it.'
+                "You already have a running Jupyter notebook server; "
+                'use "dask-chtc jupyter status" subcommand to see it\'s logs.'
```

Comment:
I'm wondering whether we should go the full mile and just print the status as part of this message. It's not an expensive operation if we've already done the query (we should cache the query so we don't do it twice in rapid succession). If you don't want to touch that code, let me know and I'll do it in a followup PR (leaving this as-is).

Reply:
> print the status as part of this message.

I think that's a good idea. I'd also add commands on how to stop/delete that Jupyter server.

Reply:
@stsievert We'll probably have some new users trying things out in the next few days, so I'd like to get your improvements merged. Did you want to take a crack at this, or should I merge as-is?

Reply:
I haven't had time to try anything; feel free to merge if you're satisfied.

Reply:
And we can always add a follow-on PR.

Reply:
:+1: